src/components/CmsZone.vue

```diff
@@ -1,80 +1,98 @@
 <template>
-  <div
-    v-infinite-scroll="{ action: next, enabled: isScrolling }"
-    :class="{ 'scrollable-content': isScrolling, 'cms-zone--inspect': isInspectOverlayEnabled }"
-  >
-    <button
-      v-if="isInspectOverlayEnabled"
-      class="cms-zone__zone-label"
-      @click.stop.prevent="shouldShowInspectModal = true"
-    >
-      {{ zoneId }}
-    </button>
-
-    <cms-inspect-sheet
-      v-model="shouldShowInspectModal"
+  <div>
+    <!-- Search Bar -->
+    <cms-search v-if="displaySearch()" @updateSearchQuery="updateSearchQuery($event)">
+      <template #close-icon>
```

Comment:
nit: I think maybe you can do this? 
```
<template v-slot="close-icon" #close-icon  />
```
(If this is just a different version of [vue](https://vuejs.org/v2/guide/components-slots.html#Named-Slots), no worries, just a little simpler)

Reply:
#close-icon is just short hand for v-slot:"close-icon"